lib/services/usagereporter.go

```diff
@@ -263,6 +263,25 @@ func (u *UsageUIRecoveryCodesPrintClick) Anonymize(a utils.Anonymizer) prehogv1.
 	}
 }
 
+// UsageCertificateIssuedEvent is an event emitted when a certificate has been
+// issued, used to track the duration and restriction.
+type UsageCertificateIssuedEvent prehogv1.CertificateIssuedEvent
```

Comment:
Should these newtypes have the `Event` suffix or not? It's about an even split right now, we should pick one.

Reply:
I've removed it at least from this new event.